### ISSUE TITLE
Display for a specific item becomes invisible after equipping a weapon

### ISSUE THREAD

**Issue by GiraPomba** [SlavicPotato/ied-dev#52]
This is a weird issue that I've no idea why it's happening. 
So far, some specific items from a mod cause this. I've tried looking at the mesh files to see if anything was different, but to no avail.

Perhaps you can take a look if you have some time?
The "issue" is that the display becomes invisible if the player equips a weapon (so far, only happens with weapons). The display still says that the item is loaded tho.

The mod in question, [https://www.nexusmods.com/skyrimspecialedition/mods/49532](https://www.nexusmods.com/skyrimspecialedition/mods/49532) , which requires Campfire.

How to reproduce:
- Create a custom display for the player in Display Manager.
- Select Single and load the Form: xx02d82e Backpack, White Fur (Zenithar).
- Set the NPC Root node (or any other node).
- This will load the ground model for the backpack.
- Equip a weapon and the display becomes invisible (the straps and amulet attached to the backpack are still visible).

Image for reference:
[https://i.imgur.com/COrmQEt.png](https://i.imgur.com/COrmQEt.png)

**Comment by SlavicPotato** [comment]
Skyrim has an obscure backpack visual display system for weapons similar to scabbards - if a weapon mesh has a node named 'Backpack' it'll attempt to move it from the weapon mesh onto the skeleton, then clean it up after it's unequipped - it does this by looking up the node by name on the skeleton and removes it if found. 

Notice the name of that disappearing mesh in those nifs..

tl;dr - no nodes named 'Backpack' may be attached above npc root (from ied or otherwise) or they'll get removed, they need to be renamed in the mesh

**Comment by GiraPomba** [comment]
Lol, wasn't aware of that. Makes sense now.
I created a mod to "reposition" backpacks using the ground model while making the armor-addon itself invisible via .esp changes. I attached the custom display to a node using skeleton extender.

I guess I have to include meshes with the blocks adjusted then. Didn't want to do this, but if there's no other way... 

Off topic: Btw I sent you a an anination file. You might find it interesting.
